README.md

```diff
@@ -178,3 +178,56 @@ bucket into a publicly browsable file tree. Useful if you want something
 like Apache's directory index for your software download repository.
 Plume release handles this as well, so it does not need to be run as part of
 the release process.
+
+## Cloud Provider Credentials
```

Comment:
This is perhaps a nit, but the section nesting places this at the same level as an individual mantle tool.  I realize that this is non-trivial to fix.

Mantle calls these "platforms".

Reply:
> This is perhaps a nit, but the section nesting places this at the same level as an individual mantle tool.

That was intentional. Multiple tools use this, so it doesn't belong under any one in particular. The best way to avoid this is to refactor this README into a smaller README and a `docs/` directory, but I don't know if thats beyond the scope of this PR.

> Mantle calls these "platforms".
👍

Reply:
Right, I meant that a better organization would be:

# Mantle
## Tools
### gangue
### kola
## Credentials
### aws